mmdet/ops/nms/nms_wrapper.py

```diff
@@ -154,7 +169,7 @@ def batched_nms(boxes, scores, idxs, nms_cfg, class_agnostic=False):
             boxes_for_nms = boxes + offsets[:, None]
 
     nms_type = nms_cfg_.pop('type', 'nms')
-    nms_op = eval(nms_type)
+    nms_op = get_nms_from_type(nms_type)
```

Comment:
What about `ast.literal_eval`?

Reply:
As I understand, `ast.literal_eval` will not work in this situation. Here is the information from the [documentation](https://docs.python.org/3/library/ast.html#ast.literal_eval):
> The string or node provided may only consist of the following Python literal structures: strings, bytes, numbers, tuples, lists, dicts, sets, booleans, None and Ellipsis.

So, `ast.literal_eval` cannot process `nms` function.